SpotifySearch/SpotifySearch/AlbumTableViewController.swift

```diff
@@ -0,0 +1,125 @@
+//
+//  AlbumTableViewController.swift
+//  SpotifySearch
+//
+//  Created by Edward Anchundia on 10/28/16.
+//  Copyright © 2016 Edward Anchundia. All rights reserved.
+//
+
+import UIKit
+
+class AlbumTableViewController: UITableViewController {
+
+    internal let albumTableViewCellIdentifier: String = "AlbumCellIdentifier"
+    internal var albums: [Album] = []
+    internal let albumEndPoint: String = "https://api.spotify.com/v1/search?q=blue&type=album&limit=50"
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        self.getAlbums(apiEndPoint: albumEndPoint) { (albums: [Album]?) in
+            if albums != nil {
+                    
+                DispatchQueue.main.async {
+                    self.albums = albums!
+                    self.tableView.reloadData()
+                    
+                }
+            }
+        }
+        
+    }
+    
+    func getAlbums(apiEndPoint: String, callback: @escaping ([Album]?) -> Void) {
```

Comment:
This function should be in it's own class file like APIHelper.swift or something. Follow M-V-C